### DISCUSSION TITLE
Guess 🇺🇸 state

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#861] [Galaxy Trivia] [answered]
🍎☕🌧️
🌵🔬👽
🦡🧀🍺

**Comment by peterbaccaro** [comment]
1. **New York** 
2. **Arizona**
3. **Wisconsin**

**Reply by ParanoidUser** [reply]
Nice try! Tiny hint: 🍎 are really edible. There are 51 of them 👽

**Comment by PatilHarshh** [comment]
1. Oregon
2. New Mexico
3. Wisconsin

**Reply by ParanoidUser** [reply]
Close!🔥But, OR is not really in the 🍎 business.

**Comment by PatilHarshh** [comment, marked as answer]
1. Washington
2. New Mexico
3. Wisconsin

**Reply by ParanoidUser** [reply]
Bingo! 👏
